.github/workflows/test-objc.yaml

```diff
@@ -119,11 +119,15 @@ jobs:
   unit-tests-carthage:
     name: Unit tests (Carthage)
     runs-on: macos-latest
+    env:
+      GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - name: Check out code
         uses: actions/checkout@v2
       - name: Pull Carthage dependencies
-        run: carthage bootstrap
+        run: |
+          carthage bootstrap --use-xcframeworks
+          carthage build --no-skip-current --use-xcframeworks
```

Comment:
Might be nice to split this into two steps, like in `project-carthage`, at least for consistency.